viewer/webpack.config.js

```diff
@@ -140,7 +141,15 @@ module.exports = env => {
             });
           });
         }
-      }
+      },
+      new CopyWebpackPlugin({
+        patterns: [
+          {
+            from: './packages/tools/src/Measurement/styles/',
+            to: './packages/tools/src/Measurement/styles/'
```

Comment:
As far as I can tell, this folder only contains a css file. Checkout the spinner in the core package for inspiration. It also uses css, but instead of copying to the build folder it bundles the css. We have configured webpack to use a raw-loader for .css files. So you should be able to import it normally `import measurementToolStyle from './PATH/TO/myCss.css';`.

You should then be able to add it do the dom element with something like:
`myElement.appendChild(document.createTextNode(measurementToolStyle));`

At least this is how it is done for the spinner.